### ISSUE TITLE
Customizing Django Admin

### ISSUE THREAD

**Issue by Sajeyks** [section-io/engineering-education#1082]
### Brief Summary: 
The Django framework comes with its own admin templates.With some know-how its possible to customize it into suiting your project even more.

### Key Takeaways: 
1. How to access the Django admin page.
2. Identifying some of the features that can be  customized.
3. Customizing the Django admin to suit their project.

### References: 
N/A

**Comment by ninjaginja** [comment]
good topic @Sajeyks  - approved. Please be mindful of readability/grammar as you're composing your article. We do not provide editing services and require that all articles be polished for grammar/readability prior to submitting a draft for review. If in doubt, we suggest having one of your peers review the draft prior to opening a PR. Thank you.

**Comment by Sajeyks** [comment]
@ninjaginja  does the readability concern have anything to do with my PR that is yet to find a reviewer?

**Comment by ninjaginja** [comment]
No, that comment was solely based on the description that you provided in this issue. Our team of reviewers is working through the backlog of articles as quickly as possible. Thanks for your patience.

**Comment by Sajeyks** [comment]
Thanks for your time

**Comment by hectorkambow** [comment]
@Sajeyks just wanted to check if there was any update on this article topic

**Comment by Sajeyks** [comment]
I will be submit the PR b4 Sunday.
